AppendBlobReader/src/consumer.js

```diff
@@ -342,53 +324,46 @@ async function appendBlobStreamMessageHandlerv2(context, serviceBusTask) {
         );
         var blockBlobClient = containerClient.getBlockBlobClient(serviceBusTask.blobName);
 
-        context.log.verbose(`Download blob content, offset: ${serviceBusTask.startByte}, count: ${serviceBusTask.startByte + batchSize - 1}, option: ${JSON.stringify(options)}`);
+        context.log.verbose(`Downloading blob, rowKey: ${serviceBusTask.rowKey}, offset: ${serviceBusTask.startByte}, count: ${serviceBusTask.startByte + batchSize - 1}, option: ${JSON.stringify(options)}`);
         let downloadBlockBlobResponse = await blockBlobClient.download(serviceBusTask.startByte, batchSize, options);
-        await streamToBuffer(context, downloadBlockBlobResponse.readableStreamBody, serviceBusTask).then(
-            async function (value) {
-                context.log.verbose("Successfully downloaded data, sending to SUMO.");
-                await sendDataToSumoUsingSplitHandler(context, value, sendOptions, serviceBusTask).then(async (dataLenSent) => {
-                    await releaseLockfromOffsetTable(context, serviceBusTask, dataLenSent).then(function () {
-                        context.done();
-                    });
-                }).catch(function (err) {
-                    context.log.error(`Error in sendDataToSumoUsingSplitHandler: ${serviceBusTask.rowKey} err ${err}`);
-                    context.done();
-                });
-            },
-            async function (error) {
-                let discardError = errorHandler(error, serviceBusTask, context);
-                if (error !== undefined && (error.code === "BlobNotFound" || error.statusCode == 404)) {
-                    // delete the entry from table storage
-                    archiveIngestedFile(serviceBusTask, context);
-                } else {
-                    await releaseLockfromOffsetTable(context, serviceBusTask).then(function () {
-                        if (discardError) {
-                            context.done();
-                        } else {
-                            context.log.error("Failed to download data in streamToBuffer");
-
-                            // after 1 hr lock automatically releases
-                            context.done(err);
-                        }
-                    });
-                }
-            }
-        );
-        context.log(`RequestId - ${downloadBlockBlobResponse.requestId}, statusCode - ${downloadBlockBlobResponse._response.status}`);
-
+        bufferData = await streamToBuffer(context, downloadBlockBlobResponse.readableStreamBody, serviceBusTask)
+        context.log.verbose(`Successfully downloaded data, sending to SUMO. RequestId - ${downloadBlockBlobResponse.requestId}, statusCode - ${downloadBlockBlobResponse._response.status}`);
```

Comment:
`Finished Downloading numChunks: ${numChunks} dataLen: ${Buffer.byteLength(buffer, 'utf-8')} rowKey: ${serviceBusTask.rowKey}`.  why do we need to print requestid and status code if there is no error it will always be 200 right?